### ISSUE TITLE
Runner executes calls with disabled parameters

### ISSUE THREAD

**Issue by jhelbig** [a85/POSTMan-Chrome-Extension#3361]
## App Details:
```
Postman for Linux
Version 5.1.3
linux 4.11.12-200.fc25.x86_64 / x64
```

## Issue Report:
 1. Did you encounter this recently, or has this bug always been there:
    - This just noticed the runner was failing tests after updating to 5.1.3
 2. Expected behaviour:
    - Runner executes and applies tests to calls, ignoring any of the disabled or unchecked parameters

## Steps to reproduce the problem
 1. create a new folder and api call
 2. enter some parameters to send but disable(un-check) one of them
 3. save
 4. open Console to view calls being made
 5. execute runner for the test api call you just made
 6. review console and compare to what is being sent vs what should be sent.

---
Again, the only reason why I noticed this is because my tests failed due to invalid or unexpected values being sent.

**Comment by prashantagarwal** [comment]
@jhelbig Hey, can you provide some screenshot of the request you are running and Postman console. It will help us debug this issue faster.

**Comment by jhelbig** [comment]
sure thing.  So I re-created it by setting a param on the get call, disabled it by un-checking it.  and ran the runner for the collection.  The collection had the un-checked param in the request.

![screenshot from 2017-08-19 00-07-09](https://user-images.githubusercontent.com/11377461/29483702-699adc08-8472-11e7-8cef-fa2a3e3a1696.png)

**Comment by kunagpal** [comment]
@jhelbig This should have been fixed in the 5.2.0 release, feel free to reopen if your issue persists. Thanks! 😄

**Comment by kunagpal** [comment]
This has been fixed in v5.3.0 of the app, do go ahead and check it out 😄
